advocacy_docs/partner_docs/CommVaultGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,108 @@
+6. To enable deduplication on the storage, move the Use deduplication toggle key to  the right, and in the Deduplication DB location box, browse to select the path to  the deduplication database.
+7. To move to the next setup option, click Save
+
+<p align="center">
+  <img  alt="Setup Storage Pool/Disk Storage" src="Images/DiskStorageConf.png">
+</p>
+
+8. Creating a Server Backup Plan in Core Setup: In order to create a server backup plan in Core Setup, define where the data is stored and how often must the data undergo a backup.
+9. In the Plan name box,type the name of the plan e.g EPAS Server Plan.
+10. In the Backup destinations section, select the Storage and Retention Period.
+11. In the RPO section, select the Backup frequency and Start time for the backup to start as per the backup frequency.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/BackupPlanConf.png">
+</p>
+
+### Setup EDB Postgres Advanced Server
+1. From the navigation pane, go to Protect > Databases.
+2. Click Add server.
+3. Select the database type for EDB Postgres Advanced server, which in this case is PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd1.png">
+</p>
+
+4. In the Database server name box, enter the server name.
+5. In the Username and Password boxes, enter the credentials to connect to the server.
+6. From the Plan list, select the server plan created in step 8 under Core Server Wizard section for use with EDB Postgres Advanced Server.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd2.png">
+</p>
+
+7. Once the server is added, a Job will run to install client on the Server. The screen shots below display the process from Job creation to software installation.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd3.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd4.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd5.png">
+</p>
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/ServerAdd6.png">
+</p>
+
+### Setup EDB Postgres Advanced Server Instance
+1. From the navigation pane, go to Protect > Databases > DB Instances.
+2. Click Add instance, and then select PostgreSQL.
+
+<p align="center">
+  <img  alt="Creating a Server Backup Plan in Core Setup" src="Images/CreateInstance1.png">
+</p>
+
+3. From the Server name list, select a server where you want to create the new instance.
+4. In the Instance Name box, type the EDB Postgres Advanced Server instance name.
+5. From the Plan list, select the server plan created in step 8 under Core Server Wizard section for use with EDB Postgres Advanced Server.
+6. Under Connection details, enter the following information.
```

Comment:
Replace 
"EDB Postgres Advanced Server" with
with
"EDB database"
for the next three entries.